driver_author_analysis.py

```diff
@@ -26,7 +26,7 @@
     generate_hyperedge_distribution(nodelist_filename, edgelist_filename, headers_filename, foldername)
     generate_keyword_digest(mbox_filename, output_filename=foldername+"/author_keyword_digest.txt", author_uid_filename=author_uid_filename,
                             json_filename=headers_filename, top_n=250, console_output=False)
-    generate_author_ranking(headers_filename, output_filename=foldername+"/tables/author_ranking.csv", active_score=2, passive_score=1)
+    get(headers_filename, output_filename=foldername+"/tables/author_ranking.csv", active_score=2, passive_score=1)
```

Comment:
I suggest using `from lib.analysis.author import ranking` and then `ranking.get` as many other libraries will also have `get` function making the code less readable and increasing the possibility for future errors.